bcda/cclf/cclf_test.go

```diff
@@ -570,6 +570,14 @@ func (s *CCLFTestSuite) TestCleanupCCLF() {
 	testUtils.ResetFiles(s.Suite, BASE_FILE_PATH+"cclf/")
 }
 
+func (s *CCLFTestSuite) TestIsSuppressionFile() {
+	suppressionFilePath := BASE_FILE_PATH + "cclf_Suppression/T#EFT.ON.ACO.NGD1800.DPRF.D181120.T1000009"
```

Comment:
Does `cclf_suppression` make sense?  Is this suppression file considered a CCLF file?